code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -120,9 +129,11 @@ Difficulty: Very Hard
 	else
 		move_to_delay = initial(move_to_delay)
 
-	if(health <= maxHealth / 10 && final_available) //One time use final attack
+	if(health <= maxHealth / (enraged? 10 : 9) && final_available) //One time use final attack. Want to make it not get skipped as much on base colossus, but a little easier to skip on enraged as it can be used multiple times
```

Comment:
This comment is long, could you perhaps move the part you added onto another line?
```suggestion
	if(health <= maxHealth / (enraged ? 10 : 9) && final_available) //One time use final attack. Want to make it not get skipped as much on base colossus, but a little easier to skip on enraged as it can be used multiple times
```